[2021-12-25 11:21:56,269] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: survey.myfunction 2021-12-25T11:21:51.468909+00:00 [queued]>
[2021-12-25 11:21:56,433] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: survey.myfunction 2021-12-25T11:21:51.468909+00:00 [queued]>
[2021-12-25 11:21:56,448] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-12-25 11:21:56,462] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-12-25 11:21:56,463] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-12-25 11:21:56,557] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): myfunction> on 2021-12-25T11:21:51.468909+00:00
[2021-12-25 11:21:56,583] {standard_task_runner.py:52} INFO - Started process 54 to run task
[2021-12-25 11:21:56,613] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'survey', 'myfunction', '2021-12-25T11:21:51.468909+00:00', '--job-id', '185', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/survey.py', '--cfg-path', '/tmp/tmphr3va268', '--error-file', '/tmp/tmpk_ibavfo']
[2021-12-25 11:21:56,617] {standard_task_runner.py:77} INFO - Job 185: Subtask myfunction
[2021-12-25 11:21:56,850] {logging_mixin.py:104} INFO - Running <TaskInstance: survey.myfunction 2021-12-25T11:21:51.468909+00:00 [running]> on host e1668ec19f6d
[2021-12-25 11:21:57,100] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=survey
AIRFLOW_CTX_TASK_ID=myfunction
AIRFLOW_CTX_EXECUTION_DATE=2021-12-25T11:21:51.468909+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-12-25T11:21:51.468909+00:00
[2021-12-25 11:21:58,296] {python.py:118} INFO - Done. Returned value was: None
[2021-12-25 11:21:58,346] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=survey, task_id=myfunction, execution_date=20211225T112151, start_date=20211225T112156, end_date=20211225T112158
[2021-12-25 11:21:58,755] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-12-25 11:21:58,901] {local_task_job.py:146} INFO - Task exited with return code 0
